tests/test_cp_apr.py

```diff
@@ -238,12 +250,18 @@ def test_cpapr_pqnr(capsys):
     M, _, _ = ttb.cp_apr(sptensorInstance, 2, algorithm="pqnr")
     capsys.readouterr()
     assert np.isclose(M.full().data, ktensorInstance.full().data, rtol=1e-01).all()
-    M, _, _ = ttb.cp_apr(sptensorInstance, 2, algorithm="pqnr", precompinds=False)
+    M, _, output = ttb.cp_apr(sptensorInstance, 2, algorithm="pqnr", precompinds=False)
     capsys.readouterr()
     assert np.isclose(M.full().data, ktensorInstance.full().data, rtol=1e-01).all()
 
+    # Assert that params from previous run can be provided as input
+    M, _, output2 = ttb.cp_apr(sptensorInstance, 2, **output["params"])
+    capsys.readouterr()
+    assert output["params"] == output2["params"]
+
     # Edge cases
     # Confirm timeout works
+    np.random.seed(123)
```

Comment:
This seed confirms reproducibility for a given machine but not across different machines. Does it make more sense to just dump an explicit initial guess we know doesn't have an error at the first step? Was the bad line search actually making this test irrelevant or just creating noise in the pytest output? Because it looks like we still captured the time limit exceeded which is the goal of the test anyway.

If its just reducing the print out can we just catch/squash the print. If the previous test was fundamentally broken and the tensor is small can we just use an explicit initial value that doesn't hit the line search issue?

Reply:
Suggestion: using an initial guess of all ones (weights and factor_matrices) is a typical choice for deterministic testing.

Reply:
I added fixtures for two tensor problems with their solutions as well as two initial guess ktensors. Finding the right combo of problems/initial guesses that ensured PQNR threw a bad L-BFGS error made for a somewhat ugly solution (i.e., 2 tensor inputs & 2 initial guesses), but this ensures that functionality is flexed.